Source/Data/01 - openXDA.sql

```diff
@@ -3905,6 +3936,72 @@ CREATE TABLE AlarmLog
 )
 GO
 
+CREATE TABLE LatestAlarmLog
+(
+    ID INT IDENTITY(1, 1) NOT NULL PRIMARY KEY,
+    AlarmID INT NOT NULL UNIQUE REFERENCES Alarm(ID),
+    AlarmLogID INT NOT NULL REFERENCES AlarmLog(ID),
+    StartTime DATETIME NOT NULL,
+    EndTime DATETIME NULL
+)
+GO
+
+CREATE NONCLUSTERED INDEX IX_LatestAlarmLog_AlarmLogID
+ON LatestAlarmLog(AlarmLogID ASC)
+GO
+
+CREATE TRIGGER AlarmLog_UpdateLatestAlarmLog
+    ON AlarmLog
+    AFTER INSERT
+AS
+BEGIN
+    SET NOCOUNT ON;
+
+    -- Ignore logs with AlarmFactorID
```

Comment:
Why are we ignoring Anything with  `AlarmFactorID`? Assuming this was original behavior?

Reply:
Yeah, it's based on the behavior of the `AlarmGroupView`. I didn't see any comments or logic that would tip me off as to why, and I don't really even remember what `AlarmFactorID` is for, so I just kept it in.

Reply:
It looks like `AlarmFactor` is basically just a multiplier applied to the alarm threshold value. I don't understand why that exists, though. And it has its own `SeverityID`, which doesn't even get used anywhere. Do you think we need it? Perhaps we can just get rid of it.